### PR TITLE
Trademarks

### PR DESCRIPTION
**Pull request by tasha-m-k**

### CODE REVIEW

**Review by cxharris — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # zowe-cli-cics-deploy-plugin
 
-This project provides a plug-in for [Zowe CLI](https://github.com/zowe/zowe-cli) to deploy applications developed on a workstation to IBM CICS Transaction Server for z/OS (CICS). It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS. It will also provide low-level commands for performing individual steps of the deployment process that could be used as part of a CI/CD pipeline.
+This project provides a plug-in for [Zowe CLI](https://github.com/zowe/zowe-cli) to deploy applications developed on a workstation to IBM® CICS® Transaction Server for z/OS® (CICS). It aims to provide an experience similar to deploying to a cloud platform when deploying to CICS®. It will also provide low-level commands for performing individual steps of the deployment process that could be used as part of a CI/CD pipeline.
```
Comment (markcocker) [inline README.md]: I thought only the first instance of a registered trademark text needed to have the ® mark. I see two instances marked here. Also applies to other files.

File: docs/_data/sidebars/cdp_sidebar.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ entries:
     output: web
     folderitems:
 
-    - title: Zowe CLI CICS deploy plug-in
+    - title: Zowe CLI CICS® deploy plug-in
```
Comment (markcocker) [inline docs/_data/sidebars/cdp_sidebar.yml]: Should the side navigation bar have any registered marks? I see this as equivalent to the knowledge centre “Table of Contents” nav bar and it does not have any marks.

File: docs/pages/cdp/cdp-Common-Errors.md [hunk 3]
```diff
@@ -55,15 +55,15 @@ If you experience errors while using the cics-deploy plug-in, check the followin
 *Suggested actions:*
 
 * Confirm that the `--scope` and `--cicsplex` settings are correct.
-* Ask your CICS Systems Programmer to check that the CMAS for the current CICSPlex is alive and that CPSM is working properly.
+* Ask your CICS® Systems Programmer to check that the CMAS for the current CICSPlex® is alive and that CPSM is working properly.
```
Comment (markcocker) [inline docs/pages/cdp/cdp-Common-Errors.md]: Suggest "current CICSPlex"  > "cicsplex"

File: docs/pages/cdp/cdp-Creating-Zowe-CLI-profiles.md [hunk 4]
```diff
@@ -116,19 +116,19 @@ zowe profiles create cics-deploy-profile --help
...
 | Option | Description |
 | --- | --- |
-| cics-plex | CPSM CICSplex name. This will typically be set to the same as cicsplex in the cics-deploy profile. |
-| region-name | The name of the CICS region name to interact with. |
-| protocol | HTTP or HTTPS to use to connect to the CICS WUI server. |
-| host | Host name of the CICS WUI server. |
-| port | Port number of the CICS WUI server. |
-| user | User ID to identify yourself to the CICS WUI server . |
-| password | Password to identify yourself to the CICS WUI server. |
+| cics-plex | CPSM CICSplex® name. This will typically be set to the same as cicsplex in the cics-deploy profile. |
```
Comment (markcocker) [inline docs/pages/cdp/cdp-Creating-Zowe-CLI-profiles.md]: Is “CICSplex” (note casing)  a registered trademark? For example https://www.ibm.com/support/knowledgecenter/SSGMCP_5.5.0/configuring/parallel-sysplex/sysplex_cicsplex_synergies.html does not show a mark on the first instance off “CICSplex” but does for first instance of “CICSPlex®”

File: docs/_config.yml [hunk 5]
```diff
@@ -4,10 +4,10 @@ output: web
 # this property is useful for conditional filtering of content that is separate from the PDF.
 # note that this adaption of Tom Johnston's Jekyll documentation theme does not generate PDF outputs
 
-topnav_title: Zowe CLI CICS deploy plug-in
+topnav_title: Zowe CLI CICS® deploy plug-in
```
Comment (markcocker) [inline docs/_config.yml]: Is trademarking page titles necessary? I don't see this being done in the CICS KC.

File: docs/pages/cdp/cdp-Log-and-trace-files.md [hunk 6]
```diff
@@ -60,6 +60,6 @@ They can all be useful when identifying the cause of an error, especially STDERR
 
 ### JES job logs
 
-If you are familiar with the z/OS environment, you can often find some useful diagnostics by accessing the JES job log associated with the CICS region you deployed to.  The `cics-deploy push` command outputs the job IDs for all the CICS regions in the scope after a deployment.  The MSGUSR, SYSPRINT, SYSOUT and JESMSGLG spool files can all provide useful information.   If you are not so fluent with the world of z/OS, you may need to discuss your needs with one of your organization's CICS system administrators.
+If you are familiar with the z/OS® environment, you can often find some useful diagnostics by accessing the JES job log associated with the CICS® region you deployed to.  The `cics-deploy push` command outputs the job IDs for all the CICS® regions in the scope after a deployment.  The MSGUSR, SYSPRINT, SYSOUT and JESMSGLG spool files can all provide useful information.   If you are not so fluent with the world of z/OS, you may need to discuss your needs with one of your organization's CICS® system administrators.
```
Comment (markcocker) [inline docs/pages/cdp/cdp-Log-and-trace-files.md]: Can you pls remove double and tripple spaces after full stops. I realise HTML will collapse them but they are unnecessary.

**Review by markcocker — CHANGES_REQUESTED** [review]
Do instances of "Java" need to be "Java™"?
I see headings and sub-headings having (R) marks, but should they be?
Also see review comments.

**Review by markcocker — APPROVED** [review]
Thanks. Changes look good.

**Pull request merged**